website/content/docs/agent/telemetry.mdx

```diff
@@ -89,6 +89,17 @@ These are some metrics emitted that can help you understand the health of your c
 
 **What to look for:** For a healthy cluster, you're looking for a `lastContact` lower than 200ms, `leader` > 0 and `candidate` == 0. Deviations from this might indicate flapping leadership.
 
+### Certificate authority expiration
+
+| Metric Name                            | Description                                                             | Unit    | Type  |
+| :------------------------- | :---------------------------------------------------------------------------------- | :------ | :---- |
+| `consul.mesh.active-root-ca.expiry` | The number of seconds until the root CA expires. Consul updates the value every hour. | seconds | gauge |
+| `consul.mesh.active-signing-ca.expiry` | The number of seconds until the signing CA expires. Consul updates the value every hour. | seconds | gauge |
```

Comment:
Optional: Consider linking to how to find out what the intermediate TTL configuration is for the CA system

Reply:
Maybe to this doc?  https://developer.hashicorp.com/consul/docs/connect/ca/consul#intermediatecertttl